setup.py

```diff
@@ -23,6 +23,7 @@
         "types-PyYAML",  # NOTE: Needed due to mypy typeshed
         "types-requests",  # NOTE: Needed due to mypy typeshed
         "types-pkg-resources",  # NOTE: Needed due to mypy typeshed
+        "types-SQLAlchemy>=1.4.49",
```

Comment:
- [ ] Include comment why it is needed. Use the same comment as the other `types-` prefixed lint deps